styles.css

```diff
@@ -661,7 +661,63 @@ p {
   margin-bottom: 15px;
 }
 
+.footer{
+  height : 50px;
+  background-color: rgb(197, 209, 222);
+}
 .footer p {
-  color: #9c9c9c;
+  color: grey;
   padding-right: 8px;
+  margin: 10px;
+}
+.footer a:hover{
+  color : rgb(196, 24, 179);
+}
```

Comment:
Let's don't use this color on GitHub icon as it doesn't fit well with the whole theme of the website. We can just scale on hover without changing the color.